scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -852,7 +867,7 @@ def handle(self):
 
         logger.debug("Running %s (%s)" % (self.name, self.type_str))
         command = [self.generator_cmd]
-        command += ["-C", self.build_dir, "run"]
+        command += ["-C", domain_build, "run"]
```

Comment:
when not using sysbuild, domain_build does not fit here, we should avoid having sysbuild terminology become the default, this will be confusing.

Reply:
Updated to use `build_dir` instead of `domain_build` for the variable name